docs/src/pages/[platform]/getting-started/migration/migration.vue.mdx

```diff
@@ -3,8 +3,6 @@ import { Alert } from '@aws-amplify/ui-react';
 
 ## 2.x to 3.x
 ### Installation
-For Vue.js 2 users please continue to use our legacy library found [here](https://github.com/aws-amplify/amplify-ui/tree/legacy/legacy/amplify-ui-vue).
```

Comment:
We will deprecate these packages, IMO we should remove this instruction.

Reply:
ok!